tools/debian/adjust-for-release

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Can you add a general comment to this file as to its intent?

The name of the file is pretty descriptive, but maybe you can add info on when it should be called.